src/python/pants/task/testrunner_task_mixin.py

```diff
@@ -408,8 +408,8 @@ def register_options(cls, register):
     # https://github.com/pantsbuild/pants/issues/5073
 
     register('--fast', type=bool, default=True, fingerprint=True,
-             help='Run all tests in a single pytest invocation. If turned off, each test target '
-                  'will run in its own pytest invocation, which will be slower, but isolates '
+             help='Run all tests in a single invocation. If turned off, each test target '
+                'will run in its own invocation, which will be slower, but isolates '
```

Comment:
I think that this indentation might break things. The below line's quote starts at the same space as the top line's quote, which might confuse Python.

Either way, for consistency with the repo's code style, it would be best to add some extra spaces to this line so that the quotes line up on all three lines.

Thanks!

Reply:
another artifact of web editing. Lesson learned: next time clone the repo and use a real editor.